src/common/client/users.client.ts

```diff
@@ -0,0 +1,6 @@
+import axios from "axios";
+import { Response } from "../types/Response.types";
+
+export const fetchUsers = () => {
+  return axios.get<Response.GetUsers>("https://demo4780686.mockable.io/users");
```

Comment:
I created an endpoint that provides user-info.

Getting the information with "axios" & "redux-thunk"